packages/local_auth/local_auth/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-## NEXT
+## 2.1.3
 
 * Updates minimum Flutter version to 2.10.
+* Update the `intl` constraint to ">=0.17.0 <0.19.0" (0.18.0 isn't really a breaking change).
```

Comment:
This is more implementation information than we need in the CHANGELOG, which is for client-facing information. I would just say "Adds compatibility with `intl` 0.18.0." (for all four instances).

Reply:
Seems better to me too. Done.